_includes/integrations/vue-tech-ref.md

```diff
@@ -0,0 +1,367 @@
+
+**Default value:** `" "`
+
+**Type:** String
+
+##### Example: `initial-value`
+
+```xml
+<editor
+  initial-value="Once upon a time..."
+/>
+```
+
+#### `inline`
+Used to set the editor to inline mode. Using `<editor :inline=true />` is the same as setting `{inline: true}` in the TinyMCE selector (`tinymce.init`).
+
+For information on inline mode, see: [User interface options - `inline`]({{site.baseurl}}/configure/editor-appearance/#inline) and [Setup inline editing mode]({{site.baseurl}}/general-configuration-guide/use-tinymce-inline/).
+
+**Default value:** `false`
+
+**Possible values:**  `true`, `false`
+
+##### Example: `inline`
+
+```xml
+<editor
+  :inline=true
+/>
+```
+
+#### `modelEvents`
+
+Sets the trigger events for [v-model events](#forminputbindingsv-model).
+
+For a list of available TinyMCE events, see: [Editor events]({{site.baseurl}}/advanced/events/#editorevents).
+
+**Default value:** `"change keyup undo redo"`.
+
+**Type:** String
+
+##### Example: `modelEvents`
+
+```xml
+<editor
+  modelEvents="change keydown blur focus paste"
+/>
+```
+
+#### `plugins`
+Used to include plugins for the editor. Using `<editor plugins="lists code" />` is the same as setting `{plugins: 'lists code'}` in the TinyMCE selector (`tinymce.init`).
+
+For information on adding plugins to TinyMCE, see: [Add plugins to TinyMCE]({{site.baseurl}}/plugins/).
+
+**Type:** String or Array
+
+##### Example: `plugins`
+
+```xml
+<editor
+  plugins="lists code"
+/>
+```
+
+#### `tag-name`
+Only valid when [`<editor :inline=true />`](#inline). Used to define the HTML element for the editor in inline mode.
+
+**Default value:** `div`
+
+**Type:** String
+
+##### Example: `tag-name`
+
+```xml
+<editor
+  :inline=true
+  tag-name='myTextArea'
+/>
+```
+
+#### `toolbar`
+Used to set the toolbar for the editor. Using `<editor toolbar="bold italic" />` is the same as setting `{toolbar: 'bold italic'}` in the TinyMCE selector (`tinymce.init`).
+
+For information setting the toolbar for TinyMCE, see: [User interface options - toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).
+
+**Default value:** `" "`
+
+**Possible values:**  See [Editor control identifiers - Toolbar controls]({{site.baseurl}}/advanced/editor-control-identifiers/).
+
+**Type:** String
+
+##### Example: `toolbar`
+
+```xml
+<editor
+  plugins="code"
+  toolbar="bold italic underline code"
+/>
+```
+
+### Form Input Bindings: `v-model`
+
+The `v-model` directive can be used to create a two-way data binding. For example:
+
+```html
+<editor v-model="content"></editor>
+```
+
+For information on `v-model` and form input bindings, see: [Vue.js documentation - Form Input Bindings](https://vuejs.org/v2/guide/forms.html).
+
+### Event binding
+
+Functions can be bound to editor events, such as:
+
+```html
+<editor @onSelectionChange="handlerFunction"></editor>
+```
+
+Where the `handlerFunction` is triggered with the event and returns an object containing two properties:
```

Comment:
Same as in the tinymce-angular PR - maybe "and is called with two arguments" or something similar instead of "and returns an object containing two properties". Because it is `(e, editor) => void`, right?

Reply:
updated